hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbReaderTest.cpp

```diff
@@ -378,7 +378,7 @@ class ServiceHootApiDbReaderTest : public CppUnit::TestFixture
     member = relationMembers.at(1);
     HOOT_STR_EQUALS("w1", member.role);
     CPPUNIT_ASSERT_EQUAL((long)1, member.getElementId().getId());
-    CPPUNIT_ASSERT_EQUAL(3, relation->getTags().size());
+    CPPUNIT_ASSERT_EQUAL(2, relation->getTags().size());
```

Comment:
One less tag because the `type` tag doesn't exist in the `OsmMap` in memory because the `type` is in the member variable and doesn't need to be duplicated in a tag.